ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAP-423 More precise DWP check reset

## Description of change
There is no need to reset NINO/DWP check unless the provider changes attributes that have an impact on this DWP check. These attributes are only `last name`, and `DoB` (and obviously NINO on the follow-up page).

Do not reset NINO unless last name or DoB change.

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-423

## Notes for reviewer

## How to manually test the feature
Before, NINO was reset with any change on the client details. Now it should only be reset if first name or DoB are changed.
It reduces instances were provider is fixing typos that are not related to DWP check.
